jdk/test/java/awt/KeyboardFocusmanager/DefaultPolicyChange/DefaultPolicyChange_Swing.java

```diff
@@ -21,15 +21,14 @@
  * questions.
  */
 
-/*
-  @test
-  @bug 6741526
-  @summary KeyboardFocusManager.setDefaultFocusTraversalPolicy(FocusTraversalPolicy) affects created components
-  @library ../../regtesthelpers
-  @build Sysout
```

Comment:
Please confirm that we do not need "@build Sysout" here

Reply:
I confirm that we do not need "@build Sysout", but we need "@library ../../regtesthelpers"

Reply:
If you run just this test(so no other tests compile this library) it will fail due to compilation error, since Sysout class from the library is used at line 86. The patch from JDK10 did not delete it.

Reply:
add '@build Sysout' and del '8004693'.